Libraries/Azure.psm1

```diff
@@ -2849,6 +2849,14 @@ function Add-AzureAccountFromSecretsFile {
 }
 
 Function Upload-AzureBootAndDeploymentDataToDB ($DeploymentTime, $AllVMData, $CurrentTestData) {
+	# Get the Database data
+	$dataSource = $Global:XMLSecrets.secrets.DatabaseServer
+	$dbuser = $Global:XMLSecrets.secrets.DatabaseUser
+	$dbpassword = $Global:XMLSecrets.secrets.DatabasePassword
+	$database = $Global:XMLSecrets.secrets.DatabaseName
+	if (!($dataSource -and $dbuser -and $dbpassword -and $database)) {
```

Comment:
Only Add pre-check for '$dataSource -and $dbuser -and $dbpassword -and $database' , this can speed up the deployment and testing, when LISAv2 user does not provide Database information in the secret file. If LISAv2 user does not want uploading Boot and Deployment Data to DB, just skip this additional post-deployment action.